### ISSUE TITLE
Existing tweens in View immediately stop when new tween is started

### ISSUE THREAD

**Issue by benjaminchlee** [MaximeCordeil/IATK#3]
Calling TweenPosition() (as a result of changing dimensions) in View.cs stops and freezes any other active tweens in other views.

Steps to reproduce:
1. Create multiple IATK visualisations
2. Configure them such that they have visible marks/geometry
3. In a single frame, update a dimension for all IATK visualisations
4. Only the last updated visualisation will be animated. The other visualisations will appear unchanged (axes are fine) as their tweens have been interrupted.

This causes the visualisations to be "delayed", such that changing their dimensions will always cause the marks to be in the positions of the previously set configuration, not the newest one.

**Comment by benjaminchlee** [comment]
The last updated visualisation also has its marks/geometry increase in size during the animation, despite no values relating to the size changing.

**Comment by atzanteotl** [comment]
I've pushed a fixed for the freezing tweens. Please see if that fixes the issue.

I haven't fixed the size issue yet. Does that only occur when leaving and entering runtime?

**Comment by benjaminchlee** [comment]
The fix seems to have done the trick! Animations work as expected in parallel and no size issues seem to be present as a result of this.

There is a new issue #4 that the update seems to have caused however.
